config/config.exs

```diff
@@ -28,13 +28,21 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:messages_api_user, :user_id]
 
+config :logger,
+  backends: [:console, Sentry.LoggerBackend]
+
 config :ueberauth, Ueberauth,
   providers: [
     cognito: {SignsUi.Ueberauth.Strategy.Fake, []}
   ]
 
 config :phoenix, :json_library, Jason
 
+config :sentry,
+  enable_source_code_context: true,
+  root_source_code_path: File.cwd!(),
```

Comment:
Should this not be `root_source_code_paths: [File.cwd!()]`? I don't see a "singular" version of this option in the documentation.

Reply:
I got that code from the docs it presented me when I created the project.

Reply:
Ah, it appears the singular version still works, [but is deprecated](https://github.com/getsentry/sentry-elixir/blob/7a1ba52c1ffd1c3b0371f9a00ed172e4cfabca74/lib/sentry/config.ex#L91).